tests/feature_selection/test_selection.py

```diff
@@ -3,28 +3,54 @@
 # Maximilian Christ (maximilianchrist.com), Blue Yonder Gmbh, 2016
 
 import numpy as np
-from unittest import TestCase
 import pandas as pd
```

Comment:
why have you deleted the test class?
I always like the classes, as they summarize things quite nicely.